default-config.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// This allows you to run the data-service module independently with environment variables set.
+require('dotenv').config();
+
+module.exports = {
+  dataService: {
+    model: process.env.DATASERVICEMODEL ? process.env.DATASERVICEMODEL : 'postgres',
+  },
+  postgres: {
+    user: process.env.PGUSER ? process.env.PGUSER : 'test',
```

Comment:
Something like this might be neater:
```suggestion
    user: process.env.PGUSER || 'test',
```